docs/core/runtime-config/garbage-collector.md

```diff
@@ -709,13 +709,21 @@ Project file:
 
 ## Standalone GC
 
-- Specifies the name of a GC native library that the runtime loads in place of the default GC implementation. This native library needs to reside in the same directory as the .NET runtime (**coreclr.dll** on Windows, **libcoreclr.so** on Linux).
+- Specifies the name of a GC native library that the runtime loads in place of the default GC implementation. This native library needs to reside in the same directory as the assembly contained the `Main` method or as the .NET runtime (**coreclr.dll** on Windows, **libcoreclr.so** on Linux).
 
 | | Setting name | Values | Version introduced |
 | - | - | - | - |
 | **runtimeconfig.json** | N/A | N/A | N/A |
-| **Environment variable** | `COMPlus_GCName` | *string_path* | .NET Core 2.0 |
-| **Environment variable** | `DOTNET_GCName` | *string_path* | .NET 6 |
+| **Environment variable** | `COMPlus_GCName` | *string_name* | .NET Core 2.0 |
+| **Environment variable** | `DOTNET_GCName` | *string_name* | .NET 6 |
+
+- Specifies the full path of a GC native library that the runtime loads in place of the default GC implementation. To be secure, this location should be protected from potentially malicious tampering.
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | N/A | N/A | N/A |
+| **Environment variable** | `COMPlus_GCPath` | *string_path* | .NET 9 |
```

Comment:
```suggestion
```

Reply:
I don't think we need to include this prefix since no-one would have needed to use it previously, correct? I.e. it never existed before the DOTNET prefix version existed?